src/App.tsx

```diff
@@ -15,71 +16,101 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export type Props = {};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export type State = {
+  selectedGood: string;
+};
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export class App extends Component<Props, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  goodSelected = (item: string) => {
```

Comment:
Still [1 point from checklist](https://github.com/mate-academy/react_goods-selector/blob/master/checklist.md)

```suggestion
  selectGood = (item: string) => {
```